### ISSUE TITLE
Start creating intermediaries for non-obfuscated classes/methods/fields

### ISSUE THREAD

**Issue by liach** [FabricMC/intermediary#15]
We have seen situations where mojang suddenly deobfuscates existing fields/methods, breaking our mod usage. This has defeated the intermediary names' purpose, i.e. to allow mods to work across versions more easily. Also, see #9; mojang has changed obfuscation status of classes as well, making such intermediary inclusion more necessary.

**Comment by modmuss50** [comment]
Yes, I think this needs to be done. We might need a way to re-populate the obf names when yarn builds if it doesn’t have a replacement name for them.

**Comment by asiekierka** [comment]
The problem is that Mojang and other developers may introduce libraries and software which expect these de-obfuscated names to exist. We need to handle that as well.

**Comment by Player3324** [comment]
I don't think there's any realistic use case where keeping the original deobf names is beneficial, everything but calling main will most certainly need to access obfuscated parts as well..

**Comment by liach** [comment]
Now it becomes a bigger problem as seen in 20w10a where many realms classes have been moved away from deobfuscation.
